docs/cog_guides/downloader.rst

```diff
@@ -0,0 +1,547 @@
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog installversion <repo_name> <revision> <cogs>
+
+**Description**
+
+Install a cog from the specified revision of given repo.
+
+Example:
+    - ``[p]cog installversion Broken-Repo e798cc268e199612b1316a3d1f193da0770c7016 cog_name``
+
+**Arguments**
+
+- ``<repo_name>`` The name of the repo to install cogs from.
+- ``<revision>`` The revision to install from.
+- ``<cogs>`` The cog or cogs to install.
+
+.. _downloader-command-cog-list:
+
+""""""""
+cog list
+""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog list <repo_name>
+
+**Description**
+
+List all available cogs from a single repo.
+
+Example:
+    - ``[p]cog list 26-Cogs``
+
+**Arguments**
+
+- ``<repo_name>`` The repo to list cogs from.
+
+.. _downloader-command-cog-listpinned:
+
+""""""""""""""
+cog listpinned
+""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog listpinned 
+
+**Description**
+
+List currently pinned cogs.
+
+.. _downloader-command-cog-pin:
+
+"""""""
+cog pin
+"""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog pin <cogs>
+
+**Description**
+
+Pin cogs - this will lock cogs on their current version.
+
+Examples:
+    - ``[p]cog pin defender``
+    - ``[p]cog pin outdated_cog1 outdated_cog2``
+
+**Arguments**
+
+- ``<cogs>`` The cog or cogs to pin. Must already be installed.
+
+.. _downloader-command-cog-reinstallreqs:
+
+"""""""""""""""""
+cog reinstallreqs
+"""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog reinstallreqs 
+
+**Description**
+
+This command will reinstall cog requirements and shared libraries for all installed cogs.
+
+Red might ask user to use this when it clears contents of lib folder
+because of change in minor version of Python.
+
+.. _downloader-command-cog-uninstall:
+
+"""""""""""""
+cog uninstall
+"""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog uninstall <cogs>
+
+**Description**
+
+Uninstall cogs.
+
+You may only uninstall cogs which were previously installed
+by Downloader.
+
+Examples:
+    - ``[p]cog uninstall 26-Cogs defender``
+    - ``[p]cog uninstall Laggrons-Dumb-Cogs say roleinvite``
+
+**Arguments**
+
+- ``<cogs>`` The cog or cogs to uninstall.
+
+.. _downloader-command-cog-unpin:
+
+"""""""""
+cog unpin
+"""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog unpin <cogs>
+
+**Description**
+
+Unpin cogs - this will remove update lock from cogs.
+
+Examples:
+    - ``[p]cog unpin defender``
+    - ``[p]cog unpin updated_cog1 updated_cog2``
+
+**Arguments**
+
+- ``<cogs>`` The cog or cogs to unpin. Must already be installed and pinned.
+
+.. _downloader-command-cog-update:
+
+""""""""""
+cog update
+""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cog update [cogs...]
+
+**Description**
+
+Update all cogs, or ones of your choosing.
+
+Examples:
+    - ``[p]cog update``
+    - ``[p]cog update defender``
+
+**Arguments**
+
+- ``<cogs>`` The cog or cogs to update. If omitted, all cogs are updated.
```

Comment:
Should this be using square brackets? Is this a mistake of the auto-generator?

Reply:
Swapped to square brackets and reviewed all other bracket to make sure the match the `usage`